### ISSUE TITLE
Simulator not loading on Nightly

### ISSUE THREAD

**Issue by mykmelez** [mozilla/r2d2b2g#518]
The Simulator (both the 3.0 stable release and the tip of master) has stopped loading on Nightly, throwing two of the following when Nightly is started from the command line:

```
error: r2d2b2g: An exception occurred.
Component returned failure code: 0x80520012 (NS_ERROR_FILE_NOT_FOUND) [nsIXPCComponents_Utils.import]
Traceback (most recent call last):
  File "resource://gre/modules/NetUtil.jsm", line 137, in 
    aCallback(pipe.inputStream, aStatusCode, aRequest);
…
  File "resource://r2d2b2g-at-mozilla-dot-org/r2d2b2g/lib/main.js", line 19, in 
    const Gcli = require('gcli');
  File "resource://gre/modules/commonjs/sdk/loader/cuddlefish.js", line 133, in 
    result = load(loader, module);
```

cc: folks who might know if there's been a breaking change recently to the way modules are located and loaded: @ochameau @gozala @canuckistani

**Comment by ochameau** [comment]
It's not an sdk issue, but just a URL change for a devtool jsm.
Give me a minute and I'll submit a pull request against gcli module.

**Comment by ochameau** [comment]
https://github.com/erikvold/gcli-jplib/pull/1

**Comment by ochameau** [comment]
Actually, there is still something misleading in the SDK, the stacktrace isn't really helpfull.
It looks like loader code is printing the wrong stacktrace.

**Comment by mykmelez** [comment]
We fixed this in d272d1405ed7b3ffdb956439abf7b4f119f84ee0, and the fix is in the latest preview build ([Windows](https://ftp.mozilla.org/pub/mozilla.org/labs/r2d2b2g/r2d2b2g-windows.xpi), [Mac](https://ftp.mozilla.org/pub/mozilla.org/labs/r2d2b2g/r2d2b2g-mac.xpi), [Linux](https://ftp.mozilla.org/pub/mozilla.org/labs/r2d2b2g/r2d2b2g-linux.xpi)).
